python/paddle/fluid/layers/tensor.py

```diff
@@ -442,6 +443,56 @@ def argmax(x, axis=0):
     return out
 
 
+def argsort(input, axis=-1):
```

Comment:
1. Need unit testing in https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/tests/unittests/test_layers.py

2.  Better add `name`.

```
def argsort(input, axis=-1, name=None):
```

Reply:
Done